### PR TITLE
Add Entity Relationship Diagram

### LINKED ISSUE #39: Create an entity relationship diagram
We need to give people a visual reference to how all these tables relate to each other.

### PR DESCRIPTION
**Pull request by timhodson**
Fixes #39 

Uses Mermaid JS (pulled from [unpkg.com](https://unpkg.com/).

Preview of the browser rendered diagram.
image

### CODE REVIEW

File: topics/entity-relationships.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+| `}o` | `o{` | Zero or more (no upper limit)
+| `}|` | `|{` | One or more (no upper limit)
+
+Labels on relationships indicate where there is a reliance on a particular key being present.
+
+## Attributes
+
+We have not listed all attributes of a table, only those that can be used as either primary of foreign keys.
+
+<div class="mermaid">
```
Comment (timhodson) [inline topics/entity-relationships.md:30]: This DIV is rendered by mermaid-js to become the ERD diagram.  
Things to check here are attribute/table names being correct.

File: topics/entity-relationships.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+| `|o` | `o|` | Zero or one |
+| `||` | `||` | Exactly one |
+| `}o` | `o{` | Zero or more (no upper limit)
+| `}|` | `|{` | One or more (no upper limit)
+
+Labels on relationships indicate where there is a reliance on a particular key being present.
+
+## Attributes
+
+We have not listed all attributes of a table, only those that can be used as either primary of foreign keys.
```
Comment (lordtatty) [inline topics/entity-relationships.md:28]:
```suggestion
We have not listed all attributes of a table, only those that can be used as either primary or foreign keys.
```

**Review by lordtatty — APPROVED** [review]
Looks great! Good work. 🙂 

Just one small typo I've suggested.

File: _includes/mermaid.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<script type="text/javascript"
+  src="https://unpkg.com/mermaid@8.10.2/dist/mermaid.min.js">
+</script>
+<script type="text/javascript" 
+    src="https://unpkg.com/jquery@3.6.0/dist/jquery.min.js">
+</script>
+
+<script>
+jQuery(document).ready(function() {
```
Comment (junglebarry) [inline _includes/mermaid.html:9]: Could we drop jQuery completely?

```suggestion
document.addEventListener('DOMContentLoaded', function() {
```

**Review by junglebarry — APPROVED** [review]
LGTM!

**Pull request merged**